lib/graphql/queries.js

```diff
@@ -900,6 +900,31 @@ export const getHostPendingApplicationsQuery = gql`
   }
 `;
 
+/**
+ * A query to get a collective source payment methods. This will not return
+ * virtual cards, as a virtual card cannot be used as a source payment method
+ * for another payment method.
+ */
+export const getCollectiveSourcePaymentMethodsQuery = gql`
+  query Collective($id: Int) {
+    Collective(id: $id) {
+      id
+      paymentMethods(types: ["creditcard", "prepaid"], hasBalanceAboveZero: true) {
+        id
+        uuid
+        name
+        data
+        monthlyLimitPerMember
+        service
+        type
+        balance
+        currency
+        expiryDate
+      }
+    }
+  }
+`;
+
```

Comment:
I think these lines are coming from a merge issue, the query doesn't wxist anymore in this file on `master`

Reply:
Yeah, I suppose I messed up in the merge 😧